node.gyp

```diff
@@ -159,8 +160,15 @@
       'type': '<(node_target_type)',
 
       'dependencies': [
-        'node_js2c#host',
-        'deps/nghttp2/nghttp2.gyp:nghttp2'
+        'node_js2c#host'
+      ],
+
+      'conditions': [
+        [ 'node_shared_nghttp2=="false"', {
+          'dependencies': [
+            'deps/nghttp2/nghttp2.gyp:nghttp2'
+          ]
+        }]
```

Comment:
Is this necessary?  It's covered by the conditional around line 712, right?

Reply:
I didn't write the conditionals for `v8_enable_inspector==1`, but I assumed that the author requires all bundled libraries in that case, so requiring the bundled nghttp2 as with the other bundled libraries seemed proper.

Reply:
Ah, sorry, I mean line 254 in node.gypi.